testing.rst

```diff
@@ -535,6 +535,15 @@ This allows you to create all types of requests you can think of:
     :ref:`framework.test <reference-framework-test>` option is enabled).
     This means you can override the service entirely if you need to.
 
+.. caution::
+
+    Before each request, the client reboots the kernel, recreating the container from scratch.
+    This ensures that every requests are "isolated" using "new" service objects.
+    Also, it means that entities loaded by Doctrine repositories will be "detached", so they will need to be
+    refreshed by the manager or queried again from a repository.
+    when a new request is sent (because the entity manager was "reset"). You have to query again entities if you want
```

Comment:
IMHO this line and the rest can be removed now.

Reply:
@HeahDude do you mean this one?

> when a new request is sent (because the entity manager was "reset"). You have to query again entities if you want

Reply:
Yes!